### PR TITLE
OD-500 [Fix] Users will not open detail view when they click on the bookmark icon in the Directory layout

### PR DESCRIPTION
**Pull request by YaroslavOvdii**
@sofiiakvasnevska 

## Issue
https://weboo.atlassian.net/browse/OD-500

## Description
Use `event.target` instead of the `$(this)` so we can receive the correct target

## Screenshots/screencasts
https://share.getcloudapp.com/xQubmpKW

## Backward compatibility
This change is fully backward compatible.

## Reviewers
@upplabs-alex-levchenko @AndrRyaz

### CODE REVIEW

File: js/layout-javascript/small-card-code.js [hunk 1]
```diff
@@ -334,7 +334,7 @@ DynamicList.prototype.attachObservers = function() {
 
       var $el = $(this);
 
-      if ($el.hasClass('small-card-bookmark-holder') || $el.parents('.small-card-bookmark-holder').length) {
+      if ($(event.target).hasClass('small-card-bookmark-holder') || $(event.target).parents('.small-card-bookmark-holder').length) {
```
Comment (tonytlwu) [inline js/layout-javascript/small-card-code.js:337]: Cache the jQuery variable

**Review by tonytlwu — CHANGES_REQUESTED** [review]
Rejected +1

File: js/layout-javascript/small-card-code.js (hunk 1 above)
Comment (YaroslavOvdii) [reply js/layout-javascript/small-card-code.js]: @tonytlwu Fixed

**Review by tonytlwu — APPROVED** [review]
@sofiiakvasnevska This is ready for testing

Risk: Medium
Affected: Directory layouts in LFD
Verify: Users can bookmark/unbookmark and open detail views as usual

Squash and merge: Use PR title

**Pull request merged**